### ISSUE TITLE
Don't mount /var/run/docker.sock in the docker-compose

### ISSUE THREAD

**Issue by Keyruu** [novuhq/novu#741]
## What?
Don't mount the docker socket in the docker-compose.

## Why?
I don't think mount the docker socket is even necessary for novu. Mounting the docker socket is also a security concern, because you give every container a lot of power with including the socket. Also this would be confusing for people who don't use docker, because they don't have the docker socket :D

## How?
Just remove the volume mounts from the docker-compose.yml.

## Other information
I already tried it without the mounts and everything seemed to have worked fine.

**Comment by scopsy** [comment]
Yep, don't see a problem with it. Initially it was just added because seen this in other similar setup, so included this blindly. But actually don't see why not removing this from the default template, and people might add this if they will need it regardless..

**Comment by kymppi** [comment]
I can do this, expect a PR in few hours.
